### DISCUSSION TITLE
Support for standard win32 apps?

### DISCUSSION THREAD

**Discussion by martona** [Abdelrhman-AK/WinPaletter#207] [Q&A]
Is there a way to fully apply the style to standard win32 dialogs and windows, such as the Alt+R run dialog, file properties, regedit, mmc, and so on?

**Comment by Abdelrhman-AK** [comment]
- It should be editable by Classic Colors section, but it doesn't support colorizing all Windows aspects. 
- Classic colors only changes registry values, but not values in visual styles.
- To change the remnants, you can search for a custom visual styles. Rectify11 visual styles can be an example.
